WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -625,7 +625,11 @@ final class EditableOrderViewModel: ObservableObject {
     /// - Parameter item: Item to remove from the order
     ///
     func removeItemFromOrder(_ item: OrderItem) {
-        guard let input = createUpdateProductInput(item: item, quantity: 0) else { return }
+        guard let input = ProductInputTransformer.createUpdateProductInput(item: item,
+                                                                           quantity: 0,
+                                                                           allProducts: Array(allProducts),
+                                                                           allProductVariations: allProductVariations,
+                                                                           defaultDiscount: currentDiscount(on: item)) else { return }
```

Comment:
Since we're changing the current behaviour of adding items from Orders in the app (outside the POS). Have we checked all use cases? What should we testing to confirm nothing has changed?

Reply:
@jaclync do you maybe have more context?

Reply:
This change was from moving `ProductInputTransformer` to Yosemite for code reuse with the POS use case. No behavior changes are expected, but it'd be great to check if this function `EditableOrderViewModel .removeItemFromOrder` has unit test coverage.